ONSdigital/eq-author-app
# Remove scrollbars from RichTextEditors

### What is the context of this PR?

Removes the scroll bar that is present inside RichTextEditors (RTE) around the place

### How to review 
1. Check the google sheet doc attached to the JIRA ticket
2. Find the corresponding RTE's around Author
3. Type a bunch of code into them
4. Hopefully they don't start scrolling
5. Cannot review the ReviewPage because it doesn't exist anymore 
6. The history page item RTE (when you edit a post that you've already made) had issues with expanding horizontally instead of vertically. Make sure that doesn't happen too.

**Here's a bunch of text if you just want to copy it**
pohasdpoahdaodhaopdhisfsdoadasdnjsdgibsf
pohasdpoahdaodhaopdhisfsdoadasdnjsdgibsf
pohasdpoahdaodhaopdhisfsdoadasdnjsdgibsf
pohasdpoahdaodhaopdhisfsdoadasdnjsdgibsf
pohasdpoahdaodhaopdhisfsdoadasdnjsdgibsf
